### PR TITLE
TRAFODION-1643

### PR DESCRIPTION
**Pull request by mashengchen**
after hbase-0.98.8(include 0.98.8), the instantiation process of ScanInfo(class) is different from before, which need KeepDeletedCells(enum) as parameter instead of boolean.

### CODE REVIEW

**Comment by mashengchen** [comment]
jenkins, retest

**Comment by DaveBirdsall** [comment]
Looks good to me. Can someone with experience with HBase APIs please review?

**Comment by DaveBirdsall** [comment]
Just checking: Is this ready to commit?

File: core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/regionserver/transactional/TrxTransactionState.java [hunk 1]
```diff
@@ -126,15 +130,49 @@
...
-
 	if (sb_sqm_98_1) {
 	    LOG.info("Got info of Class ScanQueryMatcher for HBase 98.1");
 	}
 	if (sb_sqm_98_4) {
 	    LOG.info("Got info of Class ScanQueryMatcher for HBase 98.4");
 	}
+	
+	
+	Class scaninfoClazz = null;
```
Comment (selvaganesang) [inline core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/regionserver/transactional/TrxTransactionState.java]: I believe scaninfoclazz should also be made static. Can you please check if the scanInfoConstructor becomes invalid if scanInfoclazz gets gc-ed.

**Comment by selvaganesang** [comment]
I will be merging this PR by end of day. Please review and send in your comments before.

**Comment by DaveBirdsall** [comment]
Looks good to me.

**Pull request merged**